package.json

```diff
@@ -61,10 +61,16 @@
     "typescript": "^4.4.4"
   },
   "dependencies": {
+    "@ethersproject/bytes": "^5.5.0",
+    "@ledgerhq/hw-app-eth": "^6.26.0",
+    "@ledgerhq/hw-transport": "^6.24.1",
+    "@ledgerhq/hw-transport-webhid": "^6.24.1",
     "axios": "^0.23.0",
     "bn.js": "^5.2.0",
+    "buffer": "^6.0.3",
```

Comment:
why?

Reply:
Dependencies required when performing tests with argent-x, should be suppressed....